packages/validator/tests/oas.real-world.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { OASValidator } from '../src/validator/OASValidator';
+import { OpenAPIV3 } from '@har-sdk/core';
+import yaml from 'js-yaml';
+import { resolve } from 'path';
+import { readFile } from 'fs';
+import { promisify } from 'util';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
```

Comment:
Namespace import is not allowed

```suggestion
import { readdirSync } from 'node:fs';
import { resolve } from 'node:path';
```